config/routes.rb

```diff
@@ -6,10 +6,13 @@
         get '/:id/revenue', to: 'revenue#show'
         get '/most_revenue', to: 'most_revenue#index'
         get '/most_items', to: 'most_items#index'
+        get '/find', to: 'search#show'
+        get '/find_all', to: 'search#index'
```

Comment:
More conventional to use search controller since find is a method reserved to find.
Search is more common.